pkg/scheduler/algorithm/predicates/csi_volume_predicate.go

```diff
@@ -134,33 +137,36 @@ func (c *CSIMaxVolumeLimitChecker) filterAttachableVolumes(
 			continue
 		}
 
-		driverName, volumeHandle := c.getCSIDriver(pvc)
```

Comment:
Please change this to V(5) for sake of consistency.

Reply:
Changed